controller/controller.go

```diff
@@ -542,19 +555,18 @@ func (c *Impl) handleErr(err error, key types.NamespacedName) {
 
 // GlobalResync enqueues into the slow lane all objects from the passed SharedInformer
 func (c *Impl) GlobalResync(si cache.SharedInformer) {
-	alwaysTrue := func(interface{}) bool { return true }
-	c.FilteredGlobalResync(alwaysTrue, si)
+	c.FilteredGlobalResync(c.GlobalResyncFilterFunc, si)
```

Comment:
I think this parameter is actually ignored since `FilteredGlobalResync` hard-codes using the GlobalResyncFilterFunc below and ignores its argument.. seems potentially confusing/wrong?